src/components/HomeHighlight/HighlightItem.css

```diff
@@ -0,0 +1,87 @@
+
+.highlight-Item{
```

Comment:
Aplicar como padrão do projeto
```suggestion
.highlight-Item {
```